Makefile

```diff
@@ -78,8 +78,8 @@ test:
 	grep -Rn "code-block::.*xml" docs; test "$$?" -ne "0"
 	@echo
 	@echo "Check xml code-block finished."
-	mkdir -p build/code
-	cd build/code && cmake -G Ninja ../../code
```

Comment:
Why was this removed?

Reply:
Ninja is not installed in the docker image anymore